cartography/intel/gitlab/members.py

```diff
@@ -0,0 +1,79 @@
+import logging
+from typing import Any
+from typing import Dict
+from typing import List
+
+import neo4j
+import requests
+from requests.exceptions import RequestException
+
+from cartography.util import make_requests_url
+from cartography.util import run_cleanup_job
+from cartography.util import timeit
+
+logger = logging.getLogger(__name__)
+
+@timeit
+def get_group_members(access_token:str,group:str):
+    url = f"https://gitlab.example.com/api/v4/groups/{group}/members?per_page=100"
```

Comment:
please add a comment in the source code to show the documentation url of gitlab

Reply:
yes,added